src/txorphanage.cpp

```diff
@@ -118,12 +119,12 @@ void TxOrphanage::LimitOrphans(unsigned int max_orphans, FastRandomContext& rng)
     LOCK(m_mutex);
 
     unsigned int nEvicted = 0;
-    static int64_t nNextSweep;
-    int64_t nNow = GetTime();
+    static NodeSeconds nNextSweep;
```

Comment:
Out of scope for this PR but this global is probably the source of instability in the txorphan harness.